provenance-implementation/README.md

```diff
@@ -0,0 +1,32 @@
+# Project's Implementation of Supply Chain Security
+
+The official recommendation of TAG Security is that projects provide documentation of their supply chain security decisions in a manner that streamlines end user engagement.
+
+Specifically, there is a need for end users to be supplied with clear direction regarding how a project's provenance may be validated.
+
+For provenance ingestion documentation to be useful, it cannot be distributed from the same place that provenance artifacts are generated. By distributing them from the same location, a compromise of the package distribution could also result in a compromise of the instructions users are following to validate that the packages have not been compromised.
+
+## Independent Distribution
+
+In this directory, TAG Security serves as an intermediary to ensure that end users have a consistent location to retrieve provenance documentation for any participating CNCF project.
+
+All CNCF projects are encouraged to automate generation of provenance artifacts to be included with every release, then create a subdirectory directory here for your users to reference. You may help point users to this location by adding a link to your file in your README.md and/or SECURITY-INSIGHTS.yml.
+
+TAG Security will make efforts to ensure that first-time contributions to these docs are only made by project maintainers. To streamline subsequent changes, new file contributions must also contain an entry to this repo's CODEOWNERS.
+
+## Documentation Format
+
+There is not currently a proven best practice for documenting the intended end user consumption of provenance artifacts, but some elements can be expected. For the sake of your readers, it is recommended that you only _cautiously_ add content beyond these elements.
+
+1. Intent
+    - What artifacts are supplied by your project?
+    - If you have a large number of provenance artifacts or non-standard approach, why?
```

Comment:
```suggestion
    - If you have a large number of provenance artifacts or unusual approach, why?
```

Maybe slightly more accurate, since afaik there's no clear "standard"?